charts/argo-cd/values.yaml

```
@@ -342,6 +342,15 @@ server:
   ##
   podLabels: {}
 
+  ## initContainer
+  initContainers: {}
```

```suggestion
  initContainers: []
```
This should be a list instead of a map.